src/components/my-roles/RoleCard/role-card.styles.js

```diff
@@ -0,0 +1,68 @@
+import { ReactComponent as checkIcon } from 'assets/icons/check.svg';
+import styled from 'styled-components';
+import colors from 'styles/colors';
+import mq from 'styles/media-query';
+
+export const Card = styled.div`
+  border-radius: 0.625rem;
+  border: 0.0625rem solid;
+  color: ${({ clicked }) =>
+    clicked ? `${colors.green}` : `${colors.dustyGrey}`};
+  border-color: ${({ clicked }) =>
+    clicked ? `${colors.green}` : `${colors.mercury}`};
+  cursor: pointer;
+  margin: 0.625rem;
+  min-height: 9.375rem;
+  padding: 1.5625rem;
+  position: relative;
+
+  ${mq.mobileWide`
+    min-height: 12.5rem;
+  `};
+
+  ${mq.desktop`
+     min-width: 25rem; 
+  `};
+
+  svg {
+    fill: ${({ clicked }) =>
+      clicked ? `${colors.green}` : `${colors.dustyGrey}`};
+  }
+`;
+
+export const Rectangle = styled.div`
+  background-color: ${({ clicked }) =>
+    clicked ? `${colors.green}` : `${colors.white}`};
+  border: ${({ clicked }) =>
+    clicked ? `none` : `0.125rem solid ${colors.dustyGrey}`};
+  color: ${colors.white};
+  height: 1.25rem;
+  position: absolute;
+  right: 1.25rem;
+  top: 0.9375rem;
+  width: 1.25rem;
+`;
+
+export const IconContainer = styled.div`
+  height: 3.125rem;
+  width: 3.125rem;
+`;
+
+export const CheckIcon = styled(checkIcon)`
+  height: 100%;
+  width: 100%;
+`;
+
+export const RoleText = styled.p`
+  font-weight: bold;
+`;
```

Comment:
Can simply be an h3, no need for this component
```suggestion
export const RoleText = styled.h3`
`;
```